internals/review_models.py

```diff
@@ -93,13 +103,26 @@ def set_approval(cls, feature_id, field_id, new_state, set_by_email):
     new_appr.put()
     logging.info('new_appr is %r', new_appr.key.integer_id())
 
+    # Write for new Vote entity.
+    new_vote = Vote(
+        id=new_appr.key.integer_id(), feature_id=feature_id, gate_id=field_id,
+        state=new_state, set_on=now, set_by=set_by_email)
+    new_vote.put()
+    logging.info('new_appr is %r', new_vote.key.integer_id())
```

Comment:
The logged string could say "new_vote"

Reply:
Ah yes, missed copy over, thanks for the catch 😅